.github/workflows/studio-unit-tests.yml

```
@@ -76,6 +77,7 @@ jobs:
           path: ./apps/studio/coverage/lcov.info
           retention-days: 1
 
+    timeout-minutes: 15
```

actionlint (1.7.12)

[error] key "timeout-minutes" is duplicated in "test" job. previously defined at line:25,col:5

(syntax-check)